extensions/amp-video-docking/0.1/test/test-amp-video-docking.js

```diff
@@ -271,13 +274,13 @@ describes.realWin('video docking', {amp: true}, (env) => {
       await docking.placeAt_(video, x, y, scale, step, transitionDurationMs);
 
       expect(
-        video.applyFillContent.withArgs(
+        applyFillContent.withArgs(
```

Comment:
IIRC stubbing modules in this way is considered less-than-ideal for reasons I don't recall (Justin has mentioned during stand-up). Could we instead `expect(videoLayerElement('...').classList).to.include('i-amphtml-fill-content')` here?

Reply:
Sure. Tip in case you ever need it:

```diff
- expect(element.classList).to.include('i-amphtml-fill-content')
+ expect(element).to.have.class('i-amphtml-fill-content')
```